baiji/s3.py

```diff
@@ -13,22 +13,13 @@
 a file from S3 into a local file.
 
 """
-import os
-import shutil
-import re
-from baiji.exceptions import InvalidSchemeException, S3Exception, KeyNotFound, BucketNotFound, KeyExists, get_transient_error_class
-from baiji.util.parallel import ParallelWorker
+from baiji.connection import S3Connection
```

Comment:
Yeah, I was kind of thinking of this PR as actively hiding things which were originally private by intention... I can see the argument both ways, but in practice I don't think this change will break any well written code.